apps/src/templates/sectionsRefresh/CurriculumQuickAssign.jsx

```diff
@@ -56,82 +56,25 @@ export default function CurriculumQuickAssign({updateSection, sectionCourse}) {
-        <div className={moduleStyles.buttonsInRow}>
-          <Button
-            id={'uitest-elementary-button'}
-            className={moduleStyles.buttonStyle}
-            text={i18n.courseBlocksGradeBandsElementary()}
-            size={Button.ButtonSize.large}
-            icon={
-              marketingAudience === MARKETING_AUDIENCE.ELEMENTARY
-                ? 'caret-up'
-                : 'caret-down'
-            }
-            onClick={e => {
-              e.preventDefault();
-              updateMarketingAudience(MARKETING_AUDIENCE.ELEMENTARY);
-            }}
-          />
-          <Button
-            id={'uitest-middle-button'}
-            className={moduleStyles.buttonStyle}
-            text={i18n.courseBlocksGradeBandsMiddle()}
-            size={Button.ButtonSize.large}
-            icon={
-              marketingAudience === MARKETING_AUDIENCE.MIDDLE
-                ? 'caret-up'
-                : 'caret-down'
-            }
-            onClick={e => {
-              e.preventDefault();
-              updateMarketingAudience(MARKETING_AUDIENCE.MIDDLE);
-            }}
-          />
-          <Button
-            id={'uitest-high-button'}
-            className={moduleStyles.buttonStyle}
-            text={i18n.courseBlocksGradeBandsHigh()}
-            size={Button.ButtonSize.large}
-            icon={
-              marketingAudience === MARKETING_AUDIENCE.HIGH
-                ? 'caret-up'
-                : 'caret-down'
-            }
-            onClick={e => {
-              e.preventDefault();
-              updateMarketingAudience(MARKETING_AUDIENCE.HIGH);
-            }}
-          />
-          <Button
-            id={'uitest-hoc-button'}
-            className={moduleStyles.buttonStyle}
-            text={i18n.courseOfferingHOC()}
-            size={Button.ButtonSize.large}
-            icon={
-              marketingAudience === MARKETING_AUDIENCE.HOC
-                ? 'caret-up'
-                : 'caret-down'
-            }
-            onClick={e => {
-              e.preventDefault();
-              updateMarketingAudience(MARKETING_AUDIENCE.HOC);
-            }}
-          />
-          <input
-            checked={decideLater}
-            className={moduleStyles.input}
-            type="checkbox"
-            id="decide-later"
-            onChange={toggleDecideLater}
-          />
-          <label className={moduleStyles.decideLater} htmlFor="decide-later">
-            {i18n.decideLater()}
-          </label>
-        </div>
+      <div className={moduleStyles.input}>
+        <label className={moduleStyles.decideLater} htmlFor="decide-later">
+          {i18n.decideLater()}
+        </label>
+        <input
+          checked={decideLater}
+          className={moduleStyles.inputBox}
+          type="checkbox"
+          id="decide-later"
+          onChange={toggleDecideLater}
+        />
+        <h3>{i18n.assignACurriculum()}</h3>
+        <h5>{i18n.useDropdownMessage()}</h5>
```

Comment:
probably outside the scope of this pr, since it was already there, but would it be a pain to change this h5 to an h4? best practice is to not skip levels, since it confuses the document structure if you're navigating by headings

Reply:
ooo good callout- I can update this! I think I only chose it because the styling looked close to the spec. I'll do this in the next pr!